src/main/java/net/minestom/server/network/packet/server/play/data/WorldPos.java

```diff
@@ -12,6 +12,7 @@ public record WorldPos(@NotNull String dimension, @NotNull Point blockPosition)
             NetworkBuffer.STRING, WorldPos::dimension,
             NetworkBuffer.BLOCK_POSITION, WorldPos::blockPosition,
             WorldPos::new);
+    public static final NetworkBuffer.Type<WorldPos> OPT_NETWORK_TYPE = NETWORK_TYPE.optional();
```

Comment:
Not a fan, I dont want to start bloating all classes network types or codecs which may be used in an optional context. Generally id only like to define a second constant if its a fundamentally different codec, otherwise the single allocation of a second optional wrapper is acceptable.